contracts/test/IStarknetMessaging.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0.
+pragma solidity ^0.6.12;
+
+import "./IStarknetMessagingEvents.sol";
+
+interface IStarknetMessaging is IStarknetMessagingEvents {
+    /**
+      Sends a message to an L2 contract.
+      Returns the hash of the message.
+    */
+    function sendMessageToL2(
+        uint256 toAddress,
+        uint256 selector,
+        uint256[] calldata payload
+    ) external returns (bytes32);
+
+    /**
+      Consumes a message that was sent from an L2 contract.
+      Returns the hash of the message.
+    */
+    function consumeMessageFromL2(uint256 fromAddress, uint256[] calldata payload)
+        external
+        returns (bytes32);
+
+    /**
+      Starts the cancellation of an L1 to L2 message.
+      A message can be canceled messageCancellationDelay() seconds after this function is called.
+      Note: This function may only be called for a message that is currently pending and the caller
+      must be the sender of the that message.
+    */
+    function startL1ToL2MessageCancellation(
```

Comment:
nice import from starknet-dai-bridge!

Reply:
Yeah, starknet is just extending their messaging system. Good things happening